src/services/user/utils/aggregations.js

```diff
@@ -145,6 +122,59 @@ const getConsentStatusSwitch = () => {
 	};
 };
 
+const getConsentStatusOk = (ageFirstLevel, ageSecondLevel) => {
+	const consentCaseOk = [
+		{
+			$and: ifConsentsApprovedAndOlderThanAgeDate(ageSecondLevel),
+		},
+		{
+			$and: ifAllConsentsApprovedAndBetweenAgeDates(ageFirstLevel, ageSecondLevel),
+		},
+		{
+			$and: ifParentsConsentsApprovedAndYoungerThanAgeDate(ageFirstLevel),
+		},
+	];
+
+	if(Configuration.get("FEATURE_TREAT_USER_AS_REGISTERED_WITHOUT_CONSENTS_ENABLED") === true) {
```

Comment:
same here, I think the feature flag name is a bit confusing....
at least WITHOUT_CONSENTS_ENABLED - where do we have consents enabled/disabled? 
is this about ignore consents or something like that?

Reply:
Info update
This ticket is put on hold due to discrepancy between registration process in documentation and real flow in application.

Response
Consents are created in user collection after registration process. And mosty after registration those initially are set up for true.  Exception is for brb where consents checkbox are put in first login so after registration consents are set to false. In admin page we see if user is registered by those parameters. So in brb, we could wrongly see that user is not registered, but in reality he might just not went for first login. My env variable WITHOUT_CONSENTS_ENABLED is not about ignoing those consents, but checking only if they are created so we know that we are after registration.